### PR TITLE
feat(base): Configure base server with health check

### PR DESCRIPTION
**Pull request by shreyasjayanna**

### CODE REVIEW

**Comment by thetonywu** [comment]
Just a note, a lot of these are vendor files that should not be checked in. Having a appropriate `.gitignore` file should address this.

**Comment by shreyasjayanna** [comment]
Thanks @tonywuhl , wasn't sure if we checked in vendor files or not.

**Review by thetonywu — APPROVED** [review]
LGTM!

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,6 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# Dependencies
+vendor/
```
Comment (thetonywu) [inline .gitignore]: You might want to consider using a more comprehensive base such as: https://www.gitignore.io/api/go,vim,macos,visualstudiocode

Comment (shreyasjayanna) [reply .gitignore]: Thanks! I'll update it in next PR.

File: pkg/health/health_test.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	"github.com/labstack/echo"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHealthRoute(t *testing.T) {
+	// Create a new echo router
+	e := echo.New()
+
+	// Regier our echo routes onto our new router
```
Comment (kienan) [inline pkg/health/health_test.go]: nit: `Register`

**Review by kienan — APPROVED** [review]
Nice, LGTM!

**Pull request merged**